srtcore/channel.cpp

```diff
@@ -878,7 +884,9 @@ srt::EReadStatus srt::CChannel::recvfrom(sockaddr_any& w_addr, CPacket& w_packet
 #ifdef SRT_ENABLE_PKTINFO
         // Without m_bBindMasked, we don't need ancillary data - the source
         // address will always be the bound address.
-        char mh_crtl_buf[sizeof(CMSGNodeIPv4) + sizeof(CMSGNodeIPv6)];
+
+        // ASAN: Moved outside of this block scope, see above.
+        //char mh_crtl_buf[sizeof(CMSGNodeIPv4) + sizeof(CMSGNodeIPv6)];
```

Comment:
```suggestion
```
Would be enough if you describe the needs in the comments. This can be simply removed.

Reply:
Update PR with this change.